src/loader-xerces.cpp

```diff
@@ -196,6 +196,18 @@ SWIDStruct XercesSWIDTagIO::load(const string & filename) {
 }
 
 
+void XercesSWIDTagIO::save(const string & filename, const SWIDStruct & what) {
+	try {
+		XMLIO::save(filename, what);
+	} catch (const XMLException & toCatch) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Catch a type that inherit from "std::exception". ([cpp:S3698](https://sonarcloud.io/organizations/openscap/rules#rule_key=cpp%3AS3698))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=openscap-libswid&pullRequest=9&issues=AWNj9DzYXOBV_YrhWXji&open=AWNj9DzYXOBV_YrhWXji)

[ISSUE_KEY]: AWNj9DzYXOBV_YrhWXji